README.md

```diff
@@ -1,14 +1,25 @@
 # react-native-pusher-push-notifications
+
 Manage pusher interest subscriptions from within React Native JS
 
-IMPORTANT!!! This module is intended to complement the default [Pusher setup](https://pusher.com/docs/push_notifications).  This module simply allows that implementation to be accessed directly from React Native JS.
+**IMPORTANT!!!**
+
+This module is intended to complement the default [Pusher Carthage setup](https://docs.pusher.com/beams/ios/sdk-integration#install-from-carthage). This module simply allows that implementation to be accessed directly from React Native JS.
```

Comment:
I would remove word `simply` as it might be simple for somebody and also not-so-simple for others.
https://jameshfisher.com/2017/02/22/dont-use-simply